src/v/raft/consensus.cc

```diff
@@ -4290,4 +4290,80 @@ size_t consensus::bytes_to_deliver_to_learners() const {
     return total;
 }
 
+ss::future<remake_learner_state_reply>
+consensus::remake_learner_state(vnode target) {
+    _probe->recovery_reset();
+    remake_learner_state_request req{
+      .node_id = _self,
+      .target_node_id = target,
+      .group = _group,
+      .term = _term};
+    vlog(_ctxlog.debug, "Issuing remake group request {}", req);
+    static constexpr auto timeout = 600s;
+    result<remake_learner_state_reply> reply
+      = co_await _client_protocol.remake_learner_state(
+        target.id(), req, rpc::client_opts(timeout));
```

Comment:
Naive question since I haven't looked into the remake_partition implementation -- does this also reset the consensus object? Or just the partition and log state?